### ISSUE TITLE
Remove references to /irods/hydroshare_vault and IRODS_CACHE_ROOT

### ISSUE THREAD

**Issue by cbcunc** [hydroshare/hydroshare#5258]
**Description of the bug**
The backing storage for the mount `/irods/hydroshare_vault` was removed from service. References to it must be removed.

Steps to reproduce the bug:
1. `./hsctl start`
2. See error

**Expected behavior**
Expected a clean Hydroshare startup

**Additional information**
```
Starting hydroshare ... error
[1B
ERROR: for hydroshare  Cannot start service hydroshare: driver failed programming external connectivity on endpoint hydroshare (ac9ded313cc51ce3a99e23496a01d747aa3378876069da3b2cae54872e3e3ee1): Bind for 0.0.0.0:15001 failed: port is already allocated
No containers to start
```

**Comment by cbcunc** [comment]
https://github.com/search?q=repo%3Ahydroshare%2Fhydroshare%20%2Firods%2Fhydroshare&type=code
and:
https://github.com/search?q=repo%3Ahydroshare%2Fhydroshare%20IRODS_CACHE_ROOT&type=code

**Comment by cbcunc** [comment]
```
./beta/deploy/hydroshare-config.yaml:IRODS_CACHE_ROOT: /irods/hydroshare_vault/nginx_tmp/beta/cache/
./beta/deploy/hydroshare-ssl-nginx.conf.template:        alias IRODS_CACHE_ROOT/;
./beta/deploy/hydroshare-config.yaml.test:IRODS_CACHE_ROOT: /irods/hydroshare_vault/nginx_tmp/beta/cache/
./beta/deploy/hydroshare-ssl-nginx.conf.template_bkp110822:        alias IRODS_CACHE_ROOT/;
./beta/hs_proxy/config/hydroshare-config.sh:IRODS_CACHE_ROOT=/irods/hydroshare_vault/nginx_tmp/beta/cache/
./beta/hs_proxy/config/hydroshare-config.sh.test:IRODS_CACHE_ROOT=/irods/hydroshare_vault/nginx_tmp/beta/cache/
./beta/nginx/config/hydroshare-config.sh:IRODS_CACHE_ROOT=/irods/hydroshare_vault/nginx_tmp/beta/cache/
./beta/nginx/config/hydroshare-config.sh.test:IRODS_CACHE_ROOT=/irods/hydroshare_vault/nginx_tmp/beta/cache/
./prod/deploy/hydroshare-config.yaml:IRODS_CACHE_ROOT: /irods/hydroshare_vault/nginx_tmp/www/cache/
./prod/deploy/hydroshare-ssl-nginx.conf.template:        alias IRODS_CACHE_ROOT/;
./prod/deploy/hydroshare-ssl-nginx.conf.template.bk:        alias IRODS_CACHE_ROOT/;
./prod/deploy/hydroshare-ssl-nginx.conf.template.bak141:        alias IRODS_CACHE_ROOT/;
./prod/deploy/hydroshare-ssl-nginx.conf.template.bak112222:        alias IRODS_CACHE_ROOT/;
./prod/deploy/hydroshare-config.yaml~:IRODS_CACHE_ROOT: /irods/hydroshare_vault/nginx_tmp/www/cache/
./prod/deploy/hydroshare-config.yaml_111422:IRODS_CACHE_ROOT: /irods/hydroshare_vault/nginx_tmp/www/cache/
./prod/deploy/hydroshare-ssl-nginx.conf.template.bak010422:        alias IRODS_CACHE_ROOT/;
```

**Comment by cbcunc** [comment]
Also need to remove `IRODS_CACHE_URI` and `irods-cache` references.

https://github.com/search?q=repo%3Ahydroshare%2Fhydroshare%20IRODS_CACHE_URI&type=code
and
https://github.com/search?q=repo%3Ahydroshare%2Fhydroshare%20irods-cache&type=code

**Comment by cbcunc** [comment]
```
./beta/deploy/hydroshare-config.yaml:IRODS_CACHE_URI: irods-cache
./beta/deploy/hydroshare-ssl-nginx.conf.template:    location /IRODS_CACHE_URI/ {
./beta/hs_proxy/config/hydroshare-config.sh:IRODS_CACHE_URI=irods-cache
./beta/nginx/config/hydroshare-config.sh:IRODS_CACHE_URI=irods-cache
./prod/deploy/hydroshare-config.yaml:IRODS_CACHE_URI: irods-cache
./prod/deploy/hydroshare-ssl-nginx.conf.template:    location /IRODS_CACHE_URI/ {
```

**Comment by devincowan** [comment]
released as 2.10.3
